burrow/test_burrow.py

```diff
@@ -0,0 +1,30 @@
+# (C) Datadog, Inc. 2010-2016
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+# stdlib
+from nose.plugins.attrib import attr
+
+# 3p
+
+# project
+from tests.checks.common import AgentCheckTest
+
+
+instance = {
+    'status_url': 'http://localhost:8000'
+}
+
+
+@attr(requires='burrow')
+class TestBurrow(AgentCheckTest):
+    """Basic Test for burrow integration."""
+    CHECK_NAME = 'burrow'
+
+    def test_check(self):
+        """
+        Testing Burrow check.
+        """
+        self.load_check({}, {})
+        self.assertTrue(True)
```

Comment:
looks like this test is not finished, let me know if you'd like some pointers on how to write it

Reply:
I would prefer to have some indications to write tests :)  
(sorry for the delay...)

Reply:
@lypnol We're working on getting Agent6 shipped now, but if you have specific questions about the test @olivielpeau is happy to assist as time permits. Alternatively, feel free to stop by Community Office hours for real-time assistance (https://github.com/DataDog/dd-agent/wiki/Community-Office-Hours). Thanks again for the contribution!